webpack.config.plugin.js

```diff
@@ -0,0 +1,95 @@
+// erquire environment setup
+require('./webpack.config.environment');
+
+const webpack = require('webpack');
+// const ExtractTextPlugin = require('extract-text-webpack-plugin');
```

Comment:
are these lines necessary?

Reply:
not necessary for now, it was a work in progress for scss parsing